src/state/persisted-state.ts

```diff
@@ -613,7 +613,7 @@ export interface IBinding {
     type: string;
     physicalId: string;
     lastCommittedHash: string;
-    partitionAccountId?: string;
+    partitionId?: string;
```

Comment:
check whether it makes sense to read the old attribute (and write the new attribute)